### DISCUSSION TITLE
Why was Glow 24.11 Nord theme removed?

### DISCUSSION THREAD

**Discussion by OpenSoftware-World** [turkaysoft/glow#37] [Q&A] [answered]
As someone who loves the Nord theme, I was saddened by its removal. Why was it removed? @roines45

**Comment by turkaysoft** [comment]
Glow policy had to be removed due to circumstances.

**Reply by OpenSoftware-World** [reply]
> Glow policy had to be removed due to circumstances.

Like What? @roines45

**Reply by turkaysoft** [reply, marked as answer]
The Nord theme is a custom theme and is not suitable for Glow use.
